setup.py

```diff
@@ -965,24 +969,17 @@ def my_new_compiler(**kw):
         return my_compiler()
     return orig_new_compiler(**kw)
 
-
 # No way to cleanly wedge our compiler sub-class in.
-from distutils import ccompiler
-from distutils._msvccompiler import MSVCCompiler
-
 orig_new_compiler = ccompiler.new_compiler
-ccompiler.new_compiler = my_new_compiler
-
-base_compiler = MSVCCompiler
```

Comment:
Doesn't look like `base_compiler` was used for anything other than being an alias.